uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/ContractEntitlementCalculationTest.php

```diff
@@ -275,6 +275,26 @@ public function testGetContractEndDateReturnsTheContractPeriodEndDate() {
     $this->assertEquals($this->contract['period_end_date'], $calculation->getContractEndDate());
   }
 
+  public function testIsCalculationUnitInHoursShouldReturnTrueIfAbsenceTypeCalculationUnitIsInHours() {
+    $absenceType = $this->prophesize(AbsenceType::class);
+    $absenceType->isCalculationUnitInHours()
+                ->shouldBeCalled()
+                ->willReturn(true);
```

Comment:
Since this functionality to mock the absence type class is used like four times already both in this test class and in the EntitlementCalculation test class, I'm wondering if we can wrap it up in a function that accepts a boolean for the return value of isCalculationUnitInHours and put the function in an helper class where it can be called.

Reply:
I've thought of that at first, but then I thought the amount of duplication is not that big (just 2 lines of code, if you ignore the formatting) and moving it to another method would actually decrease readability, as it your force the dev having to check another method to understand what is going on.